siac/parse.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+// parseSize converts strings of form 10GB to a size in bytes, if it fails an error is returned.
+func parseSize(strSize string) (string, error) {
+	// If the value is at least 2 or 3 characters it may have a suffix. To verify it could be a suffix one of the
+	// characters are checked to see it is outside the ascii range for numbers (48-57), letters are (65-90, 97-122).
```

Comment:
Needs to be wrapped to 80 chars. This one is at 110 + tab (tabs can count for 2, 4, 8, doesn't matter)